internal/controller/csiaddons/persistentvolumeclaim_controller.go

```diff
@@ -65,6 +65,7 @@ var (
 	rsCSIAddonsDriverAnnotation     = "reclaimspace." + csiaddonsv1alpha1.GroupVersion.Group + "/drivers"
 
 	krcJobScheduleTimeAnnotation = "keyrotation." + csiaddonsv1alpha1.GroupVersion.Group + "/schedule"
+	krcJobDisableAnnotation      = "keyrotation." + csiaddonsv1alpha1.GroupVersion.Group + "/enable"
```

Comment:
The name contains `Disable`, but the value ends with `/enable`. This is a little confusing.